src/index.js

```diff
@@ -15,73 +15,71 @@ const plugin = ({ term, display, actions }) => {
 
     //fall back on the ts prefix
     if (!match) {
-        match = term.match(/^tsu?\s(.*)/);
+        match = term.match(/^tsu?\s*($|.*)/);
     }
 
     if (match) {
         const isUTC = term.startsWith('tsu');
         const input = match[1];
 
-        if (input) {
```

Comment:
for me this would be enough to achieve what you want (which I wanted to do in the first place, don't know why I forgot to test it)